devices/HapticGlove/src/HapticGlove.cpp

```diff
@@ -537,22 +551,24 @@ class HapticGlove::SenseGloveImpl::SenseGloveHapticActuator : public wearable::a
 
     bool setHapticCommand(double& value) const override
     {
+        return true;
+    }
 
-        std::lock_guard<std::mutex> lock(m_gloveImpl->mutex);
+    bool setHapticsCommand(std::vector<double>& forceValue, std::vector<double>& vibrotactileValue) const override
+    {
 
-        if (!m_gloveImpl->isAvailable(m_actuatorName,
-                                      m_gloveImpl->gloveData.humanHapticActuatorNameIdMap)) {
-            yError() << LogPrefix << "The actuator name (" << m_actuatorName
-                     << ") is not found in the list of actuators.";
+        std::lock_guard<std::mutex> lock(m_gloveImpl->mutex);
 
-            return false;
+        for (size_t i = 0; i < m_gloveImpl->nFingers; i++)
+        {
+            m_gloveImpl->gloveData.fingersHapticFeedback[i] = forceValue[i];
+            m_gloveImpl->gloveData.fingersHapticFeedback[i + 5] = vibrotactileValue[i];
```

Comment:
Why not using a different vector?

Reply:
Well, because of the msg structure, I wanted to make it more readable.